localstack/services/awslambda/invocation/lambda_models.py

```diff
@@ -64,6 +64,9 @@
 SNAP_START_SUPPORTED_RUNTIMES = [Runtime.java11, Runtime.java17]
 
 
+BUCKET_ACCOUNT = "949334387222"
```

Comment:
It would be helpful to add a comment clarifying that this is just some random account

Reply:
Will add!